install.sh

```diff
@@ -56,7 +56,7 @@ curl -sL --retry 3 "${RELEASE_URL}" | tar zx --strip 1
 echo "Installing to $DESTDIR"
 install circleci "$DESTDIR"
 
-command -v circleci
+command -v "$DESTDIR"circleci
```

Comment:
We might want to remove any potential trailing `/` in DESTDIR and force a `/` before the binary name.
```suggestion
command -v "${DESTDIR%/}"/circleci
```